Objects/stringlib/unicode_format.h

```diff
@@ -1044,14 +1062,28 @@ formatteriter_next(formatteriterobject *it)
         if (conversion_str == NULL)
             goto done;
 
-        tuple = PyTuple_Pack(4, literal_str, field_name_str, format_spec_str,
-                             conversion_str);
+        Py_XINCREF(literal_str);
```

Comment:
None of these pointers is NULL, isn't?

And why increment refcounters of non-borrowed references and later decrement them?

Reply:
Fixed in 814cb3b. That's right, none of the pointers is NULL. The reason for the increment and decrement is because I was getting segfaults so I tried to use idempotent incref-decref mirroring the ones that happen in `PyTuple_Pack` and after to check if any of the segfaults was due to incorrect incref/decref. I just removed both the increfs and the decrefs and everything is OK. Sorry for the confusion.